css/index.css

```diff
@@ -71,11 +71,39 @@ p {
     margin: 0 auto;
 }
 
+/* HEADER NAV */
+.logo {
+    width: 150px
+}
+header nav.top {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    margin: 20px 0;
+}
+header nav a {
+    text-decoration: none;
+    color: black;
+}
+header nav a:last-child {
```

Comment:
Awesome use of pseudo-selectors to add the margin at the end! You can also achieve the same effect by applying padding on the nav element.